includes/wf_crm_admin_form.inc

```diff
@@ -5,6 +5,8 @@
  * Webform CiviCRM module's admin form.
  */
 
+module_load_include('inc', 'webform_civicrm', 'includes/wf_crm_webform_base');
```

Comment:
I don't think this is needed.

Reply:
Seems to be required for loading of the class constants. Without it I get an error 

```
 Fatal error: Class 'wf_crm_webform_base' not found in /repos/webform_civicrm/includes/wf_crm_admin_form.inc on line 339
```

It probably fits better in the constructor though with the other `module_load_include` statements so I'll move it there.